float32 rightEar
float32 leftEar
---
